app/views/pages/index.html.erb

```diff
@@ -2,6 +2,5 @@
   You are logged in as <%= current_user.email %><br>
   <%= link_to 'Log Out', logout_path, method: :delete %>
 <% else %>
-  <%= link_to 'Log In', new_session_path %><br>
-  <%= link_to 'Register', new_user_path %>
+  <%= link_to 'Log In', sign_in_path %><br>
 <% end %>
```

Comment:
That's smart, thanks Sunny.  I haven't even touched the views, this is all generated by "rails scaffold", it's pretty ridiculous what it gives you.  I'm trying to focus more on the api, the gem, and try to use a rails react gem.  We'll see how that goes.  Keep em comin! Thank you!